playground/terraform/applications/backend-router/main.tf

```diff
@@ -29,27 +29,30 @@ resource "google_app_engine_flexible_app_version" "backend_app_go" {
   }
 
   readiness_check {
-    path = ""
+    path = "/readiness"
   }
 
   automatic_scaling {
-    max_total_instances = 5
-    min_total_instances = 1
+    max_total_instances = 3
```

Comment:
@andreykus Perhaps in a future PR these can be terraform variables with recommended defaults.